DEVELOPER_GUIDE.md

```diff
@@ -280,29 +282,24 @@ This method can also be used to develop against the [full distribution of OpenSe
 
 ### Configure OpenSearch Dashboards for security
 
-_This step is only mandatory if you have the [`security` plugin](https://github.com/opensearch-project/security) installed on your OpenSearch cluster with https/authentication enabled._
+_This step is only needed if you want your dev environment to also start with security. To do so both the OpenSearch node and OpenSearch Dashboards cluster need to have the security plugin installed. Follow the steps below to get setup correctly._
 
-> 1. Run `export initialAdminPassword=<initial admin password>` since it's needed by the configuration script
-> 2. Run `yarn opensearch snapshot --security`
-> 3. Wait a few seconds while the plugin is installed, configured, and OpenSearch starts up.
+To startup the OpenSearch snapshot with security
 
-Then within another window. You can start:
+1. Run `export OPENSEARCH_INITIAL_ADMIN_PASSWORD=<initial admin password>` since it's needed by the configuration script
```

Comment:
Should we add description on the password requirement as I believe it no longer accepts plain alpha-numeric password and has to be stronger one?